browser/android/background_video/background_video_playback_tab_helper.cc

```diff
@@ -55,6 +104,17 @@ bool IsBackgroundVideoPlaybackEnabled(content::WebContents* contents) {
 
   return true;
 }
+
+bool IsPictureInPictureForYouTubeVideosEnabled(content::WebContents* contents) {
+  if (!base::FeatureList::IsEnabled(
+          ::preferences::features::kBravePictureInPictureForYouTubeVideos)) {
+    return false;
+  }
+
+  content::RenderFrameHost::AllowInjectingJavaScript();
```

Comment:
I don't know why `IsBackgroundVideoPlaybackEnabled` calls this internally, it only needs to be called once and can just happen in `PrimaryMainDocumentElementAvailable`